.dockerfiles/docker-entrypoint.sh

```diff
@@ -71,7 +74,7 @@ docker_setup_env() {
 
 	declare -g ALREADY_INITIALIZED
 	# look specifically for a dictory that marks the data as initialized
```

Comment:
comment now wrong, Ok comment was wrong to start with as dictory isn't a word, but now it's wronger :)

Reply:
😆